Core/HLE/sceNet.cpp

```diff
@@ -1219,21 +1226,56 @@ static int sceNetApctlGetBSSDescEntryUser(int entryId, int infoId, u32 resultAdd
 	if (!Memory::IsValidAddress(resultAddr))
 		hleLogError(SCENET, -1, "apctl invalid arg");
 
+	// Generate an SSID name
+	char dummySSID[APCTL_SSID_MAXLEN] = "WifiAP0";
+	dummySSID[6] += static_cast<char>(entryId);
+
 	switch (infoId) {
 	case PSP_NET_APCTL_DESC_IBSS: // IBSS, 6 bytes
-		Memory::WriteStruct(resultAddr, &netApctlInfo.bssid);
+		if (entryId == 0)
+			Memory::WriteStruct(resultAddr, &netApctlInfo.bssid);
+		else {
+			// Generate a BSSID/MAC address
+			char dummyMAC[ETHER_ADDR_LEN]{ static_cast<char>(entryId) };
```

Comment:
be aware that this will only set the first entry to entryId, the rest will be 0. that might be fine though. if you want all the bytes to be set to entryId, switch back to memset.

Reply:
eh.. it's not the same with memset? ugh okay, i'll change it back